src/v3/src/transformer/email/emailChallengeTransformer.test.ts

```diff
@@ -63,6 +64,12 @@ describe('EmailChallengeTransformer Tests', () => {
     transaction.availableSteps = [{ name: 'resend', action: jest.fn() }];
     const updatedFormBag = transformEmailChallenge(transaction, formBag, mockProps);
 
+    expect(loc).toHaveBeenCalledWith(
+      'oie.email.verify.alternate.magicLinkToEmailAddress',
+      'login',
+      [redactedEmail],
+    );
```

Comment:
How do we feel about these assertions? I'm torn on if we really need them or not, since they tie our tests to the translation function. e.g. if we later update to use something that isn't `loc` we have to update these assertions too.

Reply:
So i only added these here because there was no other way to verify the parameters that were passed into the `loc` function. I can totally remove them if we dont care to check at this granular of a level?

Reply:
i don't feel strongly either way right now - but i wanted to raise it in case anyone else did. i would lean towards removing these but we can also address at a later date when we get to any potential i18n migration.

Reply:
🙏  Removed. I agree, I only did it to be super granule but with integration tests it will get caught eventually